### PR TITLE
add eviction test

### PR DESCRIPTION
**Pull request by AviAvni**

### CODE REVIEW

File: tests/flow/test_expiry.py [hunk 1]
```diff
@@ -25,7 +25,37 @@ def test01_expire_graph(self):
...
+
+        # create a single node utill OOM exception raised
+        ex = None
+        while "OOM command not allowed when used memory > 'maxmemory'" not in str(ex):
+            try:
+                redis_graph_new.query("create ()")
+            except ResponseError as e:
+                ex = e
+
+        # key should have been evicted by now
```
Comment (swilly22) [inline tests/flow/test_expiry.py:55]: should we introduce a sleep here?

File: tests/flow/test_expiry.py [hunk 2]
```diff
@@ -25,7 +25,37 @@ def test01_expire_graph(self):
...
+        redis_graph_new = Graph(GRAPH_ID + "new", redis_con)
+
+        # create a single node
+        redis_graph.query("create ()")
+
+        # create a single node utill OOM exception raised
+        ex = None
+        while "OOM command not allowed when used memory > 'maxmemory'" not in str(ex):
+            try:
+                redis_graph_new.query("create ()")
```
Comment (swilly22) [inline tests/flow/test_expiry.py:51]: do we want to introduce more data within each iteration ?

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: tests/flow/test_expiry.py (hunk 1 above)
Comment (AviAvni) [reply tests/flow/test_expiry.py:55]: no

File: tests/flow/test_expiry.py (hunk 2 above)
Comment (AviAvni) [reply tests/flow/test_expiry.py:51]: no need 2 nodes is enough

**Review by swilly22 — DISMISSED** [review]

**Review by swilly22 — APPROVED** [review]

**Pull request closed without merging**